src/axom/sidre/spio/IOManager.hpp

```diff
@@ -172,19 +172,31 @@ class IOManager
    * DataStore specified by domain_path argument.
    *
    * This currently only works if the root file was created for protocol
-   * sidre_hdf5.
+   * sidre_hdf5.  This must be called after calling write().
+   *
+   * The parameters domain_path and mesh_path are related. domain_path is the
+   * path from the root of the DataStore to the domain that is being used to
+   * generate the index.  mesh_path is the path from the group that was
+   * passed into the preceding write() call to the same domain.  If write()
+   * was called using the root group of the DataStore, then domain_path and
+   * mesh_path will be identical.  Otherwise mesh_path is a sub-path of
+   * domain_path.
+   *
```

Comment:
@nselliott -- would it be possible to add a simple, non-trivial example to the doxygen comment?

Reply:
Clarification: I was asking for something like this: 
https://github.com/mfem/mfem/blob/edfb62d8c27061a0de2b495eceffcee266b3d2cb/fem/sidredatacollection.hpp#L50

(although, not necessarily as fancy).

Reply:
I have added something here.

Reply:
Thanks @nselliott -- this is perfect.